src/types/types.ts

```diff
@@ -35,11 +28,11 @@ export type Client = {
 export type Case = {
   id: string;
   status: CaseStatus;
-  type: CaseType;
+  type: string;
```

Comment:
changed this to a string and removed the CaseType type because it's dynamic now; doesn't make sense for it to be a type